supervision/dataset/formats/coco.py

```diff
@@ -97,6 +97,36 @@ def coco_annotations_to_detections(
     return Detections(xyxy=xyxy, class_id=np.asarray(class_ids, dtype=int))
 
 
+def object_to_coco(
```

Comment:
Please always include `box`, `area`, and `segmentation` in the result COCO JSON.

Reply:
I agree with that.
I will make the changes.